HISTORY.md

```diff
@@ -6,6 +6,8 @@
  * Fix potential infinite loop in CMAES
    ([#331](https://github.com/mlpack/ensmallen/pull/331)).
 
+ * Fix Softmax Regression function partial gradient test lables vector
```

Comment:
"lables vector" -> "label vector"
or perhaps simplify the entire line to: "Fix SCD partial gradient test"